win32ss/gdi/ntgdi/freetype.c

```diff
@@ -6797,6 +6797,7 @@ IntExtTextOutW(
     BOOL bNoTransform;
     DWORD ch0, ch1;
     FONTLINK_CHAIN Chain;
+    SIZE sz;
```

Comment:
I would suggest a different name for the variable (sz sounds like "szWhatever" which is zero-terminated string, usually). Maybe "spaceWidth" or something similar would be more suitable, looking at how it is used there:
https://github.com/reactos/reactos/pull/7274/files#diff-76f0b6f8507bc93463a16f4be285a29443b8a25711b82872095dc8c8011b17adR7075
`/* Get the width of the space character */`

Reply:
Changed to "spaceWidth". Thanks.